### ISSUE TITLE
Paddy creates detached symbol copy on cmd + d (in stack)

### ISSUE THREAD

**Issue by mikeozornin** [DWilliames/paddy2-beta#23]
I have a button symbol made in a vertical stack.

When I duplicate symbol using context menu or main menu — no problems.
When I duplicate symbol using cmd + d — paddy creates layer group similar to a detached button symbol.

Screencast: http://d.mikeozornin.ru/NR5FNr

Paddy 0.3
Sketch 50.2

File: https://sketch.cloud/s/grldd

**Comment by DWilliames** [comment]
🤦‍♂️ Oops. I left in a debug function I use for myself; that when command is held down and it attempts to resize a Symbol instance, it creates a detached copy for debugging purposes.

I forgot to set this release to the 'production' version — I better fix that now.

**Comment by mikeozornin** [comment]
:-)

**Comment by aparajita** [comment]
@DWilliames Any ETA on when this will be fixed?

**Comment by DWilliames** [comment]
Yeah, sorry @aparajita, I was sick for a couple of days. And didn't have a chance to work on this... I have made a bunch of other tweak, so want to make sure it's semi-stable first. Should have the next version out tomorrow.

**Comment by DWilliames** [comment]
v0.4 out now. Should be fixed. Can you please confirm 👍

**Comment by mikeozornin** [comment]
Fixed for me on the same file: https://sketch.cloud/s/grldd
